assets/js/wp-hours-widget.js

```diff
@@ -0,0 +1,54 @@
+( function ( $ ) {
+
+	var hoursWidget = {
+
+		init: function( e ) {
+
+			var date        = new Date(),
+					cur_time    = hoursWidget.getCurrentTime( date ),
+					time_blocks = wpcw_hours.schedule[ date.getDay() ].open,
+					$open_sign  = $( '.wpcw-open-sign' );
+
+			if ( time_blocks.length ) {
+
+				for ( i = 0; i < time_blocks.length; i++ ) {
```

Comment:
@EvanHerman I'm about to change the `get_schedule()` method output slightly to use a key/value pair for open/close, so this loop will need to change to:

```js
$.each( time_blocks, function( open, close ) {
```